app/controllers/lists_controller.rb

```diff
@@ -15,6 +15,10 @@ def create
     end
   end
 
+  def index
+    @lists = List.all
```

Comment:
Definitely on the to-do list, but I think I'm going to do it as a separate pull request some time in the next day or two